_includes/overview/top.html

```diff
@@ -16,11 +16,14 @@ <h1 class="c-heading -h3">LoopBack</h1>
                 <a href="/resources">Resources</a></li>
               <li class="_nav-item {% if page.item == 'contributing' %}-active{% endif %}">
                 <a href="/contributing">Contributing</a></li>
+              <li class="_nav-item">
+                <!-- Place this tag where you want the button to render. -->
+                <a class="github-button" href="https://github.com/strongloop/loopback" data-show-count="true" aria-label="Star strongloop/loopback on GitHub">Star</a>
```

Comment:
I'd change the label to `GitHub stars for strongloop/loopback`

Reply:
It doesn't really matter as users only see the icon.